simphony_ui/openfoam_boundary_conditions.py

```diff
@@ -0,0 +1,149 @@
+
+
+class SurfaceModel(HasStrictTraits):
+
+    #: The velocity boundary condition model for the surface
+    velocity_boundary_condition = Instance(VelocityBoundaryConditionModel)
+
+    #: The pressure boundary condition model for the surface
+    pressure_boundary_condition = Instance(PressureBoundaryConditionModel)
+
+    traits_view = View(
+        Item(name='velocity_boundary_condition', style='custom'),
+        Item(name='pressure_boundary_condition', style='custom'),
+    )
+
+    def _velocity_boundary_condition_default(self):
+        return VelocityBoundaryConditionModel()
+
+    def _pressure_boundary_condition_default(self):
+        return PressureBoundaryConditionModel()
+
+
+class BoundaryConditionsModel(HasStrictTraits):
+
+    #: The boundary conditions of the inlet
+    inlet_BC = Instance(SurfaceModel)
+
+    #: The boundary conditions of the outlet
+    outlet_BC = Instance(SurfaceModel)
+
+    #: The boundary conditions of the walls
+    walls_BC = Instance(SurfaceModel)
+
+    #: The boundary conditions of the front and back
+    front_and_back_BC = Instance(SurfaceModel)
+
+    traits_view = View(
+        Tabbed(
+            UItem(
+                name='inlet_BC',
+                style='custom',
+                label='Inlet'
+            ),
+            UItem(
+                name='outlet_BC',
+                style='custom',
+                label='Outlet'
+            ),
+            UItem(
+                name='walls_BC',
+                style='custom',
+                label='Walls'
+            ),
+            UItem(
+                name='front_and_back_BC',
+                style='custom',
+                label='Front and Back'
+            )
+        )
+    )
+
+    def _inlet_BC_default(self):
+        inlet_bc = SurfaceModel()
+
+        # Add default velocity values
+        inlet_bc.velocity_boundary_condition.type = 'zeroGradient'
+
+        # Add default pressure values
+        inlet_bc.pressure_boundary_condition.type = 'fixedValue'
+        inlet_bc.pressure_boundary_condition.fixed_value = 0.008
+
+        return inlet_bc
+
+    def _outlet_BC_default(self):
+        outlet_bc = SurfaceModel()
+
+        # Add default velocity values
+        outlet_bc.velocity_boundary_condition.type = 'zeroGradient'
+
+        # Add default pressure values
+        outlet_bc.pressure_boundary_condition.type = 'fixedValue'
+
+        return outlet_bc
+
+    def _walls_BC_default(self):
+        walls_bc = SurfaceModel()
+
+        # Add default velocity values
+        walls_bc.velocity_boundary_condition.type = 'fixedValue'
+
+        # Add default pressure values
+        walls_bc.pressure_boundary_condition.type = 'zeroGradient'
+
+        return walls_bc
+
+    def _front_and_back_BC_default(self):
+        front_and_back_bc = SurfaceModel()
+
+        # Add default velocity values
+        front_and_back_bc.velocity_boundary_condition.type = 'empty'
+
+        # Add default pressure values
```

Comment:
nitpicking. This comments (and the ones above) are redundant. Remember that comments should say "why" you are doing something, not "what" you are doing.